src/Database/index.js

```diff
@@ -53,6 +53,10 @@ export default class Database {
     this._actionsEnabled = actionsEnabled
   }
 
+  get(query = ''): Array<T> {
```

Comment:
this doesn’t type-check. I want to have this API, but the method needs to pass Flow checks, needs to have a rudimentary test (Database/test.js), TypeScript support would be nice (Database/index.t.ts), and a single-line comment explaining what this is

Reply:
Ok will do proper Flow checks, also will write test too.

Reply:
@rahulrana95 are you planning to finish this PR? :)